devices/tuya.js

```diff
@@ -721,6 +721,26 @@ const fzLocal = {
             return result;
         },
     },
+    homeetec_cover: {
```

Comment:
homeetec device consisted with 2 types of controls - curtain switch + 1|2 switches,
switches and additional controls (child lock, calibration, backlight) are being controlled by DP Codes while curtain control is the main device and being treated as curtain.

as mentioned in the comment, Meta.tuyaDatapoints will be handled by `tuya.fz.datapoints` instead of default functionality of tuya cover.

Reply:
an additional reason, dp code 3 in tuya cover is positionArrived while in that device it represents functionality of calibration